src/utils/DragAndDrop.js

```diff
@@ -398,6 +416,8 @@ define(function (require, exports, module) {
                 // Special-case .zip files, so we can offer to extract the contents
                 if(Path.extname(filename) === ".zip") {
                     handleZipFile(deferred, file, filename, buffer, encoding);
+                } else if(Path.extname(filename) === ".tar") {
```

Comment:
Not yet. The code I have doesn't support it.  It's something we could add later if people want it.  I suspect `.zip` will be more popular, and `.tar` will almost never get used.